### ISSUE TITLE
Adjust heading levels on separate "ask a question" and "suggest a correction" pages

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1780]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site

https://findingaids.princeton.edu/suggest/ and https://findingaids.princeton.edu/ask_a_question/

### Description of error / warning

Missing first level heading

and Skipped heading level

The page heading is currently an h2 and should be an h1

On the "ask a question" page, "quick answers" is an h6., but there are no h3-5.

### WCAG success criterion
[1.3.1 Info and Relationships (Level A)](https://webaim.org/standards/wcag/checklist#sc1.3.1)
[2.4.6 Headings and Labels (Level AA)](https://webaim.org/standards/wcag/checklist#sc2.4.6)

### WCAG conformance level
A

### Recommended fix
Make the h2s on these pages into h1s

On ask a a question, make "quick answers" an h2, and so probably also create a sr-only h2 for the primary form area.

### Tool used to find the problem
wave

### Screenshot

<img width="1591" height="535" alt="Image" src="https://github.com/user-attachments/assets/4f4b3954-d69a-4c00-8867-7a76b5bc7ca3" />
